IPA.Loader/Logging/StandardLogger.cs

```diff
@@ -124,6 +122,10 @@ internal static void Configure()
             PrintFilter = SelfConfig.Debug_.ShowDebug_ ? LogLevel.All : LogLevel.InfoUp;
             showTrace = SelfConfig.Debug_.ShowTrace_;
             syncLogging = SelfConfig.Debug_.SyncLogging_;
+            if(SelfConfig.CommandLineValues.WriteLogs && !addedFilePrinter) {
+                addedFilePrinter = true;
+                AddDefaultPrinter(new GZFilePrinter());

```

Comment:
`GZFilePrinter` is abstract, it should be `GlobalLogFilePrinter`

Reply:
My bad, I've created my changes using the web VSC to not mess up formatting and stuff

Reply:
Oh, also in this code block, I'd like a space after the `if` and open brace on a new line